### DISCUSSION TITLE
How to Implement Alternate Icon in AIR apps

### DISCUSSION THREAD

**Discussion by idzdigital** [airsdk/Adobe-Runtime-Support#2329] [Guides & Tutorials]
Hi,
I tried to implement Alternate icon for Product Page Optimization in our AIR App using Dynamic Icon ANE . 

But, we have some Questions :-
1. How can we add 2 or more icons in single Assets.car file?
2. Where to find info.plist file? how can we edit info.plist file before creating IPA?
3. Where to find Resign script? how can we edit Resign script file before creating IPA?

Thanks in Advance.

**Comment by marchbold** [comment]
I've written some fairly detailed documentation on this here: 

https://docs.airnativeextensions.com/docs/dynamicicon/adding-icons

You can use the image script I've written to create the asset catalogue with your main icon and an alternative icon (just update icon.png and icon-alt.png and launch.png to be your assets).

https://github.com/distriqt/AIR-ImageScripts

The resign script is in the documentation listed above as well.

**Comment by idzdigital** [comment]
Hi @marchbold ,
We Follow the step that are mention in Dynamic Icon ANE documentation
but in last step while resign the package we got this error (text File attached below). 

As we have some queries : -
1. which path should we add in IPA path attribute in resign script? ( as we have unzip the ipa package)
[Resign_IPA_error.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/10131869/Resign_IPA_error.txt)

Thanks in Advance

**Reply by marchbold** [reply]
Firstly I'd suggest running the script into your application directory rather than your home directory. 

Secondly you must add your certificate to keychain and use the find signing identity by running `security find-identity -v -p codesigning` You cannot pass the path to a p12 file.
